backend/dataall/cdkproxy/stacks/sagemakerstudio.py

```diff
@@ -93,6 +93,6 @@ def __init__(self, scope, id: str, target_uri: str = None, **kwargs) -> None:
             .to_string()
         )
 
-        TagsUtil.add_tags(self)
+        TagsUtil.add_tags(stack=self, model=models.SagemakerStudioUserProfile, target_type="mlstudio")
```

Comment:
[Clarification] - belongs to another PR

Reply:
It's related to modularization :) 
Before that, we had the mapping inside `TagUtil`. I had to move it out since notebooks were part of this mapping and pass it as a parameter